ewutils.py

```diff
@@ -455,8 +455,39 @@ def decaySlimes(id_server = None):
 		finally:
 			# Clean up the database handles.
 			cursor.close()
-			databaseClose(conn_info)	
-			
+			databaseClose(conn_info)
+
+""" Flag all users in the Outskirts for PvP """
+async def flag_outskirts(id_server = None):
+	if id_server != None:
+		try:
+			client = get_client()
+			server = client.get_server(id_server)
+			conn_info = databaseConnect()
+			conn = conn_info.get('conn')
+			cursor = conn.cursor();
+
+			cursor.execute("SELECT id_user FROM users WHERE id_server = %s".format(
```

Comment:
we don't want to fetch this much data so often. restrict it to players in the outskirts here in the sql query already (WHERE poi IN %s) you'll probably need to cast ewcfg.outskirts_districts to tuple() for this